### ISSUE TITLE
Apollo 8 MCC Burn Scrubbed

### ISSUE THREAD

**Issue by pikachu1992** [dseagrav/NASSP#378]
I have a problem with Apollo 8 again (1264), but I'm not sure if it's really a problem. I assume from the checklist that it is normal for MCC burn to be scrubbed. But can it happen to more than one MCC burning? In my case both burns MCC1 and MCC2 are being scrubbed. Is this normal? Could it be a bug or will my trajectory do not need these two burns?

Thank you

**Comment by jalexb88** [comment]
It probably means you burned a very good TLI ;) I would not worry about it, you will most likely get a burn pad for MCC-3 or 4. When the mission tracking detects than an MCC will be below a certain DV threshold, it will scrub the burn which means yes, your trajectory is very good.

**Comment by pikachu1992** [comment]
Hello. Thank you very much, it was what I thought, it was just to confirm :).

Best Regards.

**Comment by indy91** [comment]
It is possible that both MCC-1 and 2 are getting scrubbed, but the threshold for this (5 ft/s burn or less) is fairly low. So if you are getting a very large MCC-3 or 4 solution later on, then there still might be something wrong.

**Comment by pikachu1992** [comment]
Hello
Okay, if that happens I'll report thank you. Just one more question, since I did not have MCC1 and MCC2, I decided to update my state vector through ApolloRTCCMFD since I was not having values below 100 in CisLunar navigation. After that Cislunar navigation started to go well! There is no problem in doing this update in ApolloRTCCMFD on an MCC mission, right?

**Comment by indy91** [comment]
Yeah, no problem at all doing updates from the RTCC MFD as well. I noticed the issue you are having with a lack of state vector updates due to scrubbed burns when I implemented the MCC-H support for Apollo 10. And on the actual missions they did get state vector updates at those times, so for the Apollo 10 MCC supprt I already implemented those alternative updates. So if the burn gets scrubbed, then you are getting a SV update anyway. Just haven't gotten around to retroactively implement this for Apollo 8, but I certainly will at some point.

If you are playing around with the RTCC MFD anyway, then you might as well check the DV of the burns that were scrubbed for you. Here is the general procedure to calculate MCC-1 and MCC-2: Open RTCC MFD, click on Maneuver Targeting, then Translunar. Click once on MAN to get to the maneuver option number 2. Click on TIG and input the MCC-2 time of ignition. After a short iteration it should give you a maneuver solution. If the DV for that is below 5 ft/s then everything is good. If it is larger or it doesn't find a solution at all, then there is a problem. Burn getting scrubbed because something is going wrong, not because things are going great.

**Comment by RL77** [comment]
Hello

Thanks for the sugestion. I used the RTCC MFD in the translunar program at my current get (30h14) and got values ​​below 5 ft / s (dVC- 3.8 ft / s) as shown below:

https://puu.sh/BabyL/61b61da752.png

After that in GET 45h16 i checked the RTCC MFD in the translunar program and already had a value of dVC 6.0 ft / s:

https://puu.sh/Baltg/fbc53939b5.png

With these values ​​I figured that MCC3 had to appear if everything was correct and appeared :)!

https://puu.sh/BalPp/c14df760f0.png

The burning dVC (6.7) is concordant with the dVC obtained previously, so I think things are really going well! Tomorrow i hope to be in moon orbit!

I'll ask one more question, when I was at a GET of + - 44h I had to make Cislunar to four stars (33,37,45,42). I started to have problems when making the first star, the PGNS light started to light up and i tried several times and still  the same.Then  I noticed that all the stars were too far from the earth's horizon and consequently out of reach of the SXT field. So I decided to choose a group of 4 stars closer to the earth (23,20,24,22). I had to cislunar without problems and with good values ​​under 50. Is there or can there be a problem for doing this?

thank you

**Comment by RL77** [comment]
Ok I now i now its for Moon Horizont...right?

**Comment by RL77** [comment]
Hello
Just to say that the MCC4 occurred with a DVC of 0.7, but I did the  maneuver anyway. I've already reached the orbit of the moon. Thank you for your support.

Best regards
